devscripts/run_tests.bat

```diff
@@ -1,4 +1,4 @@
 @echo off
 
 >&2 echo run_tests.bat is deprecated. Please use `devscripts/run_tests.py` instead
-python %~dp0run_tests.py %~1
+py %~dp0run_tests.py %~1
```

Comment:
Py launcher is an optional component. It makes more sense to use `python` directly here

Reply:
```suggestion
python %~dp0run_tests.py %~1
```